mephisto/abstractions/providers/prolific/api/data_models/__init__.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
We could really use this kind of setup in the Mephisto data model. `from mephisto.data_model.agent import Agent` is definitely redundant where `from mephisto.data_model import Agent` would not be. 

Not a TODO for right now of course, but something you have me thinking about.